src/main.ts

```diff
@@ -266,23 +217,13 @@ function main(args: string[]) {
     process.exit(1);
   }
 
-  if (settings.saveTemporaries) {
-    for (let fileName of Object.keys(jsFiles)) {
-      fs.writeFileSync(fileName, jsFiles[fileName]);
-    }
+  for (let fileName of Object.keys(jsFiles)) {
+    fs.writeFileSync(fileName, jsFiles[fileName]);
```

Comment:
As far as I can tell, it obeys all the output rules of the TypeScript compiler. We're using outDir without any problems.